ipn/local.go

```diff
@@ -258,12 +268,17 @@ func (b *LocalBackend) setClientStatus(st controlclient.Status) {
 	if st.NetMap != nil {
```

Comment:
Wait, there's another `st.NetMap != nil` check (and b.mu.Lock!) from the one up on line 220? I'd never noticed that before. Can you combine them?

Reply:
I haven't yet convinced myself that the order is not significant here. Let me take a few.